source/_patterns/04-components/site-name/site-name.md

```diff
@@ -2,8 +2,7 @@
 el: .site-name
 title: Site Name
 ---
-_Component for Site Name_
 
-## Variables
-* url: [string] The site link url.
-* site_name: [string] the site name.
+__Variables:__
+* url: [string] URL of the site name.
```

Comment:
Same note here as above, although this one's a little easier to parse because a site name won't have a separate URL.